### PR TITLE
Separate pool transactions tables

### PR DESCRIPTION
**Pull request by alter-eggo**
# Description

This pr separates pool transaction tables (investment and trades) and adds investment and trade statistics cards

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/constants/pools.ts [hunk 1]
```diff
@@ -112,7 +116,9 @@ const POOLS_MAINNET: Pools = {
       '0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014'
   },
   Pagination: {
-    PerPage: 10
+    PerPage: 10,
+    PerPool: 10,
```
Comment (garethfuller) [inline src/constants/pools.ts:120]: Why do we need both of these?

File: src/services/balancer/subgraph/types.ts [hunk 2]
```diff
@@ -191,13 +191,20 @@ export interface PoolActivity {
...
+}
+
 export interface PoolSwap {
   tokenIn: string;
   tokenOut: string;
   tokenAmountIn: string;
   tokenAmountOut: string;
   timestamp: number;
   tx: string;
+  userAddress: PoolSwapUserAddress;
```
Comment (garethfuller) [inline src/services/balancer/subgraph/types.ts:205]: I don't think we need a new interface just for this, we can simplify a little bit:
```suggestion
  userAddress: {
    id: string;
  };
```

Comment (alter-eggo) [reply src/services/balancer/subgraph/types.ts]: fix

File: src/services/balancer/subgraph/entities/poolSwaps/index.ts [hunk 3]
```diff
@@ -19,6 +20,28 @@ export default class PoolSwaps {
...
+        }
+
+        return await web3Service.getEnsAvatar(poolSwap.userAddress.id);
+      })
+    );
+
+    swaps.forEach((swap: PoolSwap, index: number) => {
+      swap.ensName = ensNames[index] as string;
+      swap.ensAvatar = ensAvatars[index] as string;
+    });
```
[the remark this answers is not in the document]
Comment (alter-eggo) [reply src/services/balancer/subgraph/entities/poolSwaps/index.ts]: fix

File: src/constants/pools.ts (hunk 1 above)
Comment (alter-eggo) [reply src/constants/pools.ts:120]: in order not to mix logics of pool transaction tables and other ones

**Comment by garethfuller** [comment]
@alter-eggo As a final thing, could you remove all the components and logic to do with the stat cards you added? I know you put a lot of effort in here so I'm sorry we need to get rid of it. But it might be a while before we can efficiently get the data required to use this logic and by then we might want to do it differently.

**Pull request merged**